Publish 2025-08-19/introducing-muninn-hall (#13)
muninn-hall/muninn-hall.com-website: blog/2025-08-19-introducing-muninn-hall/index.md

---
slug: 2025-08-19/introducing-muninn-hall
title: Introducing Muninn Hall
authors: [cat]
tags: [cat, hall]
---

In this blog post I am proud to introduce Muninn Hall to the world. I will go into a bit about why we exist, who is on staff and the role of our wonderful partners.

<!-- truncate -->

## What is Muninn Hall

Muninn Hall as covered in the introduction on the home page is a community for matrix homeserver staff. We are a blend of a Sys admin community and a T&S community and want to represent expertise in both areas.

> This focus on representing expertise in both Trust and Safety and Systems administration is nicely represented I would argue in that not only does the founding staff represent both quite well but its also represented in that we have before the hall was even founded Sys Admin partnerships in place.

Reviewer: ```suggestion
This focus on representing expertise in both Trust and Safety and Systems administration is nicely represented I would argue, in that not only does the founding staff represent both quite well, but its also represented in that we have, before the hall was even founded, Sys Admin partnerships in place.
```

## Why does Muninn Hall exist?

> Why create yet another homeserver staff community? Well, a community like Muninn Hall does not exist currently I would argue but we are also yes similar to existing communities. I was approached to found this community by individuals who had come to want to mount votes of no confidence against leadership in the existing communities.

Reviewer: ```suggestion
Why create yet another homeserver staff community? Well, a community like Muninn Hall does not exist currently, I would argue, but yes, we are also similar to existing communities. I was approached to found this community by individuals who had come to want to mount votes of no confidence against leadership in the existing communities.
```

This is what lead to the creation of Muninn Hall. One key distinguishing factor we have compared to the competition is that we do not at any level aim to represent every single homeserver in the federation.

Part of why there was no confidence vote talks against other communities was due to extremely serious breaches of professional conduct standards that us who are responsible for the Trust and Safety matters for our users are held to. As is mentioned on the home page going into more detail about this would be unethical and unprofessional.

## Who forms the founding staff of Muninn Hall

> If you go thru our GitHub, you will quickly realise that my self Cat is on the staff ofc but also, I’m writing this blog post. We also have Sky on staff that has been helping with setting up a lot of the website and been a major driving force of getting Cat to actually do shit.

Reviewer: ```suggestion
If you go through our GitHub, you will quickly realise that myself, Cat, is on the staff ofc, but also, I’m writing this blog post. We also have Sky on staff, who has been helping with setting up a lot of the website and has been a major driving force of getting Cat to actually do shit.
```

Gnuxie Founding Maintainer of Draupnir is also on our staff and is ofc a resident Draupnir whisperer and a fellow mdad user.

Tulir Creator of Meowlnir and Matrix Hacker and Tinkerer of well as long as I have been on matrix or longer is the last to mention member of our founding staff. Has been invaluable in helping get our screening bot launched and is a wellspring of knowledge on how to break matrix but also what makes Synapse tick.

## The Role of our partners

> The partnership programme is meant to recognise the projects and companies that provide a great service to the members of the hall. A great example of this is how Draupnir powers a lot of the critical moderation infrastructure that us homeserver admins need these days.

Reviewer: ```suggestion
The partnership program is meant to recognise the projects and companies that provide a great service to the members of the Muninn Hall. A great example of this is how Draupnir powers a lot of the critical moderation infrastructure that us homeserver admins need these days.
```
